base/applications/mspaint/precomp.h

```diff
@@ -21,6 +21,17 @@
 #include <shellapi.h>
 #include <htmlhelp.h>
 
+#define NDEBUG
+#include <reactos/debug.h>
+
+#if DBG
+    #define __LOCATION__  LPCSTR file, INT line
+    #define __location__  __FILE__, __LINE__
+#else
+    #define __LOCATION__  /*empty*/
+    #define __location__  /*empty*/
+#endif
```

Comment:
This is quite an invasive debugging way! I would suggest to keep this PR not as something that goes into master, but as a "tool" to apply only when local debugging of Paint is needed, and that needs to be removed afterwards once debugging has reached its goals 🤔

Reply:
`__LOCATION__` and `__location__` removed.